src/SDKs/CognitiveServices/dataPlane/Search/BingWebSearch/BingWebSearch/Microsoft.Azure.CognitiveServices.Search.WebSearch.csproj

```diff
@@ -1,5 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
```

Comment:
@sudhansu88 this should be 
Please do not move/edit code above this line and it says "below"
Can you make the change in all the csproj files.